makefiles/shared.mk

```diff
@@ -77,9 +77,14 @@ next-gen-stage: ## Host online for review
 	fi
 endif
 
+ifeq ($(MUT_PREFIX),)
+SITE_URL=${URL}/${PROJECT}/${USER}/${BRANCH_NAME}
+else
+SITE_URL=${URL}/${MUT_PREFIX}
+endif
 # Intended to be called by the autobuilder as a build command after frontend build, but before mut upload
 # Staging: https://github.com/mongodb/docs-worker-pool/blob/42bd36b1f52e49c646a79474c12d299f33d774cb/src/job/stagingJobHandler.ts#L57
 # Production: https://github.com/mongodb/docs-worker-pool/blob/1a482242fa6f1463abb059884cddb2c56ba9fad9/src/job/productionJobHandler.ts#L68
 oas-page-build:
-	node ${OAS_MODULE_PATH} --bundle ${BUNDLE_PATH} --output ${REPO_DIR}/public --redoc ${REDOC_PATH} --repo ${REPO_DIR} --site-url ${URL}/${MUT_PREFIX}
-
+	echo "oas page build - MUT_PREFIX = ${MUT_PREFIX}, Project/User/Branch = ${PROJECT}/${USER}/${BRANCH_NAME}"
```

Comment:
It's usually unidiomatic to have this sort of echo in a makefile, since make prints each command as it's run. That said, if we're really sure we want to keep this, the line shoud start with an `@` character to prevent double logging

Reply:
Yep, that would be completely reasonable!